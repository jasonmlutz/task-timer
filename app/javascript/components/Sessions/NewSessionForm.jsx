import React from 'react';

function NewSessionForm() {
  return (
    <div>Login to your account</div>
  );
}

export default NewSessionForm;
