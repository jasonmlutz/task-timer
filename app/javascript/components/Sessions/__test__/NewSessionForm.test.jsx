// REACT
import React from 'react';

// TEST UTILITIES
import renderer from 'react-test-renderer';
// COMPONENTS
import NewSessionForm from '../NewSessionForm';

describe('NewSessionForm component', () => {
  describe('STATIC TESTS', () => {
    test('renders correctly from snapshot', () => {
      const tree = renderer.create(<NewSessionForm />).toJSON();
      expect(tree).toMatchSnapshot();
    });
  });
});
