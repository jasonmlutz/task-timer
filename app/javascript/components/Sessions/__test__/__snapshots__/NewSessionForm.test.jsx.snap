// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NewSessionForm component STATIC TESTS renders correctly from snapshot 1`] = `
<div>
  Login to your account
</div>
`;
